frontend/src/components/generic/LockIcon.vue

```diff
@@ -4,7 +4,14 @@
       <v-icon v-if="value" small v-on="{ dblclick: iconDblClick, ...on }">
         mdi-lock-open-variant
       </v-icon>
-      <v-icon v-else small color="grey" v-on="{ dblclick: iconDblClick, ...on }">
+      <v-icon
+        v-else
+        small
+        color="grey"
+        :title="$tc('components.generic.lockIcon.doubleClickToUnlock')"
```

Comment:
Do we really want to mix HTML title attribute with vuetify tooltips? Better we stick to one method, shouldn't we?

Reply:
Good idea. I've added it to the minor ui issue: #3144